charts/stable/prometheus/questions.yaml

```
@@ -66,7 +78,7 @@ questions:
                 description: "Enable compression of the write-ahead log using Snappy"
                 schema:
                   type: boolean
-                  default: false
+                  default: true
```

```suggestion
                  default: false

```

Maintainers veto applied to get this merged quickly